cassdegrees/api/views.py

```diff
@@ -83,14 +83,15 @@ def search(request):
 
     # Generates a query mapping for "title":"containing text" relationships
     include = {x+"__icontains": request.GET.get(x, None) for x in columns if request.GET.get(x, None)}
+    include.update({x + "__iexact": request.GET.get(x+"_exact", None) for x in columns if request.GET.get(x+"_exact", None)})
```

Comment:
```suggestion
    include.update(
        {x + "__iexact": request.GET.get(x+"_exact", None) for x in columns if request.GET.get(x+"_exact", None)}
    )
```

Making suggested change to prevent pep-8 violation